docs/shortfin/llm/user/llama_serving.md

```diff
@@ -87,6 +87,15 @@ LLama3.1 8b f16 model.
 python -m sharktank.utils.hf_datasets llama3_8B_fp16 --local-dir $EXPORT_DIR
 ```
 
+> Note: If you have the model weights as a collection of `.safetensor` files (downloaded from HuggingFace Model Hub, for example), you can use the `convert_hf_to_gguf.py` script from the `llama.cpp` repository to convert them to a single `.gguf` file.
```

Comment:
Add a link to `` `llama.cpp` repository``

https://github.com/ggerganov/llama.cpp (included in my suggestion below)